### PR TITLE
Aqara Knob H1 (Wireless) ZNXNKG02LM

### PR DESCRIPTION
**Pull request by kirovilya**
https://www.aqara.com/cn/productDetail/wireless-smart-knob-h1
https://aliexpress.ru/item/1005003153258697.html

### CODE REVIEW

File: converters/fromZigbee.js [hunk 1]
```diff
@@ -6521,6 +6521,23 @@ const converters = {
             };
         },
     },
+    aqara_knob_rotation: {
+        cluster: 'aqaraOpple',
+        type: ['attributeReport', 'readResponse'],
+        convert: (model, msg, publish, options, meta) => {
+            if (msg.data.hasOwnProperty(570)) {
+                const act = {1: 'start_rotation', 2: 'rotation', 3: 'stop_rotation'};
```
Comment (Koenkk) [inline converters/fromZigbee.js:6529]: `start_rotation` -> maybe `start_rotating` is better?

File: converters/fromZigbee.js [hunk 2]
```diff
@@ -6521,6 +6521,23 @@ const converters = {
...
+                return {
+                    action: act[msg.data[570]],
+                    rotation_angle: msg.data[558],
+                    rotation_angle_speed: msg.data[560],
+                    rotation_percent: msg.data[563],
+                    rotation_percent_speed: msg.data[562],
```
Comment (Koenkk) [inline converters/fromZigbee.js:6535]: I think all should  be prefixed with `action_` (e.g. `action_rotation_percent`)

File: converters/fromZigbee.js (hunk 1 above)
Comment (kirovilya) [reply converters/fromZigbee.js]: renamed start and end

File: converters/fromZigbee.js (hunk 2 above)
Comment (kirovilya) [reply converters/fromZigbee.js]: done

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**